sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -1579,6 +1580,49 @@ def test_lull_logging(self):
         '.*There has been a processing lull of over.*',
         'Unable to find a lull logged for this job.')
 
+class StateBackedTestElementType(object):
+  element_count = 0
+
+  def __init__(self, num_elems):
+    self.num_elems = num_elems
+    self.value = ['a' for _ in range(num_elems)]
+    StateBackedTestElementType.element_count += 1
+    # Due to using state backed iterable, we expect there is a few instances
+    # alive at any given time.
+    if StateBackedTestElementType.element_count > 5:
+      raise RuntimeError('Too many live instances.')
+
+  def __del__(self):
+    StateBackedTestElementType.element_count -= 1
+
+  def __reduce__(self):
+    return (self.__class__, (self.num_elems, ))
```

Comment:
To make the serialized form really big, you could return `(self.num_elements, 'x' * self.num_elements)`.

Might make sense to call num_elements size instead.

Reply:
updated.  I believe the motivation is to just make the element large when serialization happens, thus the actual element now does not hold a blob of data.